views/add-event.hbs

```diff
@@ -0,0 +1,50 @@
+<div class="add-eventContainer">
+  <h3 class="userAddEvent">{user name} please add event</h3>
+
+  <form class="add-event-form" action="/add-event" method="post">
+    <label>Title: </label>
+    <input class="form-control" type="text" name="name" placeholder="title" required>
+
+    <label>Picture: </label>
+    <input class="form-control" type="file" name="imageUrl" accept="image/*" placeholder="choose picture">
```

Comment:
it doesn't work to upload an image like this.

Either:
- The image url should have type="text"
- or use a tool like S3 to handle image upload.